docs/src/main/asciidoc/_migration-guide-for-6.0.adoc

```diff
@@ -0,0 +1,145 @@
+
+==== spring-cloud-azure-starter-active-directory
+
+* Please refer to <<migration-guide-introduction, introduction>> and <<migration-guide-benefits, migration benefits>> to get a whole picture of the changes in 6.0.
+
+[#dependency-spring-cloud-azure-starter-active-directory]
+===== Dependency Changes
+
+Some dependencies in the legacy artifact have been upgraded since the modern Spring Cloud Azure 6.0 library. Please update these changes to your project to fix compilation errors.
+
+.Upgrade Dependencies
+[cols="<,<,<", options="header"]
+|===
+|Legacy dependency |Modern dependency |Description
+|javax.servlet:javax.servlet-api
+|jakarta.servlet:jakarta.servlet-api
+|Required.
+|===
+
+==== spring-cloud-azure-starter-active-directory-b2c
+
+* Please refer to <<migration-guide-introduction, introduction>> and <<migration-guide-benefits, migration benefits>> to get a whole picture of the changes in 6.0.
+
+[#dependency-spring-cloud-azure-starter-active-directory-b2c]
+===== Dependency Changes
+
+Some dependencies in the legacy artifact have been upgraded since the modern Spring Cloud Azure 6.0 library.
+Please update these changes to your project to fix compilation errors.
+
+.Upgrade Dependencies
+[cols="<,<,<", options="header"]
+|===
+|Legacy dependency |Modern dependency |Description
+|javax.servlet:javax.servlet-api
+|jakarta.servlet:jakarta.servlet-api
+|Required.
+|===
+
+==== spring-cloud-azure-starter-data-cosmos
+
+The adaptation of Spring Data 3 has not been completed, Spring Cloud Azure 6.0 will not include the Spring Data Cosmos part,
+and the related API has been deleted.
+
+[#api-spring-cloud-azure-starter-data-cosmos]
+===== API Changes
+
+This section lists the removed classes from spring-cloud-azure-starter-data-cosmos +
+
+.Removed legacy class
+* com.azure.spring.cloud.autoconfigure.data.cosmos.CosmosDataAutoConfiguration
+* com.azure.spring.cloud.autoconfigure.data.cosmos.CosmosReactiveRepositoriesAutoConfiguration
+* com.azure.spring.cloud.autoconfigure.data.cosmos.CosmosReactiveRepositoriesAutoConfigureRegistrar
+* com.azure.spring.cloud.autoconfigure.data.cosmos.CosmosRepositoriesAutoConfiguration
+* com.azure.spring.cloud.autoconfigure.data.cosmos.CosmosRepositoriesAutoConfigureRegistrar
+
+==== spring-cloud-azure-starter-servicebus-jms
+
+* Please refer to <<migration-guide-introduction, introduction>> and <<migration-guide-benefits, migration benefits>> to get a whole picture of the changes in 6.0.
+
+[#dependency-spring-cloud-azure-starter-servicebus-jms]
+===== Dependency Changes
+
+Some dependencies in the legacy artifact have been upgraded since the modern Spring Cloud Azure 6.0 library.
+Please update these changes to your project to fix compilation errors.
+
+.Upgrade Dependencies
+[cols="<,<,<", options="header"]
+|===
+|Legacy dependency |Modern dependency |Description
+|org.apache.geronimo.specs:geronimo-jms_2.0_spec
+|jakarta.jms:jakarta.jms-api
+|Transitively depends on this artifact.
```

Comment:
Does this mean customer should update other related artifacts? Please make this more clear.